test/sanity/sanity_unix.go

```diff
@@ -1,6 +1,8 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: MIT
 
+//go:build unix || linux
```

Comment:
Isn't linux a subset of unix?
Why not just:
```
//go:build unix
```
?

Reply:
Weird edge case where if there is a //go:build unix not allowing a call from linux env when the original file sanity_test.go does not contain a unix build tag but call a *_unix.go file.

For some reason, it gives error 
```
null_resource.integration_test (remote-exec): 	imports github.com/aws/amazon-cloudwatch-agent-test/internal/common: build constraints exclude all Go files in /home/ec2-user/amazon-cloudwatch-agent-test/internal/common
null_resource.integration_test (remote-exec): no Go files in /home/ec2-user/amazon-cloudwatch-agent-test
╷
│ Error: remote-exec provisioner error
│ 
│   with null_resource.integration_test,
│   on main.tf line 120, in resource "null_resource" "integration_test":
│  120:   provisioner "remote-exec" {
│ 
│ error executing "/tmp/terraform_684748912.sh": Process exited with status 1
```

This change resolved the issues. I think there is something broken for this edge case in the golang compiler.